node-tests/acceptance/broccoli-test.js

```diff
@@ -42,7 +42,7 @@ describe('broccoli-template-linter', function() {
     return cleanupBuilders();
   });
 
-  it('uses provided generateTestFile to return a test file', function() {
+  it('uses provided generateTestFile to return a test file', function(done) {
```

Comment:
Returning a promise should be the same as using the `done` pattern here (AFAIK).